### PR TITLE
Awesome-books: Feature plain javascript and html

### PR DESCRIPTION
**Pull request by gtekle**
Setup gitHub actions and linters
Created basic html skeleton
Created books list container basic html skeleton in javascript
Added Book object and books object array
Created adding books function
Added remove book functionality on remove button click.
Created localStorage for books collection
Added localStorage for form data
Fixed linter errors

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (MrBrN197) [inline .vscode/settings.json:3]: - [ ] [OPTIONAL] You can add this file to your gitingore file as it is simply a personal editor settings file. 👍

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  removeBtns = document.querySelectorAll('.btn-remove');
+  saveBooks();
+}
+
+function checkRemoveButtonClickEvent() {
+  removeBtns.forEach((removeBtn) => {
+    let id;
+    removeBtn.addEventListener('click', (event) => {
+      const idStr = event.target.id;
+      id = parseInt(idStr.substring(idStr.length - 1), 10);
```
Comment (MrBrN197) [inline js/app.js]: - [ ] [OPTIONAL] Hi, your code looks clean. I notices you have declared `let id` outside the scope of the arrow function. you could simply have it inside that scope there seems to be no need for it to live outside 👍.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @gtekle & @orozCoding 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/app.js (hunk 2 above)
Comment (gtekle) [reply js/app.js]: Hello Brian,
Thank you very much for the feedback. The issues are now fixed.

Comment (MrBrN197) [reply js/app.js]: Great 👍

**Pull request merged**